src/lib.rs

```diff
@@ -40,8 +41,24 @@ pub mod convert;
 #[cfg(feature = "std")]
 use std::vec::Vec;
 
-pub use crate::types::*;
 use crate::utils::*;
+use types::*;
+
+/// Re-export of all the basic features.
+pub mod prelude {
+    pub use crate::*;
+
+    pub use crate::types::*;
+
+    #[cfg(not(feature = "std"))]
+    pub use crate::convert::*;
+
+    #[cfg(feature = "debug")]
+    pub use crate::debug;
```

Comment:
I think I'd prefer this to live in its own namespace, given it uses names like `print32`.

Reply:
Ah right, missed the lack of `::*`.